autoload/neomake/signs.vim

```diff
@@ -245,6 +263,10 @@ endfunction
 
 let s:signs_defined = 0
```

Comment:
What about (just) setting this to `1` in case `signs` is not available?
(yes, it's hacky, but just an idea)